corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -276,14 +276,6 @@ hqDefine("geospatial/js/geospatial_map", [
             $mapControlDiv.koApplyBindings(polygonFilterModel);
         }
 
-        const $saveDrawnArea = $("#btnSaveDrawnArea");
```

Comment:
like similarly, instead of removing the save area in a different commit and then adding it in a different commit was confusing. Seeing the removal of saved area in an earlier commit, made it seem like we were removing the functionality.
If the remove and addition was in the same commit, it would be possible to see that the intent is to actually move the code elsewhere or modify the implementation.